recaptcha_enterprise/snippets/noxfile_config.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default TEST_CONFIG_OVERRIDE for python repos.
+
+# You can copy this file into your directory, then it will be imported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": ["2.7", "3.7", "3.8"],
```

Comment:
Why are we ignoring 3.7 and 3.8?

Reply:
Hi @dandhlee, as per https://endoflife.date/python, 3.7 has 7 months and 3.8 have < 2 years of support.

These code samples are provided by my team and having 2 year of support felt the right way to do. Let me know, if I should revert back.

Reply:
Client library wise and end-user wise, we're still supporting 3.7 and 3.8 until they are out of life. I don't think it's fair to leave that out unless the system (recaptcha-enterprise) decided to pin to strict Python versions. Please revert.

Reply:
I understand!